src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -375,6 +375,23 @@ void Convolution::getSupportedDescriptors() {
 
     withBiases = getOriginalInputsNumber() == 3;
 
+    // should remove after binary postops performance issue resolved
+    // heuristics: if it's int8 model and it has binary post ops we will not use brgconv
+    auto tryDisableBrgInt8 = [&]() {
+        if (shouldTryBrgconv && canBeExecutedInInt8()) {
+            dnnl::primitive_attr attrs;
+            setPostOps(attrs, MemoryDescUtils::makeDummyShape(getOutputShapeAtPort(0)).getStaticDims(), false);
```

Comment:
Just wondering is it possible to avoid redundant calls of setPostOps? Just a proposal: lets call setPostOps here and save attrs as a field, that should allow to drop setPostOps calls in initSupportedPrimitiveDescriptors and initDescriptor

Reply:
Updated.